src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,428 @@
+
+If different from PNG, TIFF or OME-TIFF, the original unprocessed data in the native format MAY be
+stored in the [`/sourcedata` directory](../02-common-principles.md#source-vs-raw-vs-derived-data).
+
+Future versions may extend this list of supported file formats.
+For example with the [OME-NGFF](https://ngff.openmicroscopy.org/latest/) format currently
+developed by OME as a successor to OME-TIFF for better remote sharing of large datasets.
+
+### Modality suffixes
+Microscopy data currently support the following imaging modalities:
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Name**                                      | `suffix` |
+| --------------------------------------------- | -------- |
+| Transmission electron microscopy              | TEM      |
+| Scanning electron microscopy                  | SEM      |
+| Micro-CT                                      | uCT      |
+| Bright-field microscopy                       | BF       |
+| Dark-field microscopy                         | DF       |
+| Phase-contrast microscopy                     | PC       |
+| Differential interference contrast microscopy | DIC      |
+| Fluorescence microscopy                       | FLUO     |
+| Confocal microscopy                           | CONF     |
+| Polarized-light microscopy                    | PLI      |
+| Coherent anti-Stokes Raman spectroscopy       | CARS     |
+| 2-photon excitation microscopy                | 2PE      |
+| Multi-photon excitation microscopy            | MPE      |
+| Super-resolution microscopy                   | SR       |
+| Nonlinear optical microscopy                  | NLO      |
+| Optical coherence tomography                  | OCT      |
+| Selective plane illumination microscopy       | SPIM     |
+
+### Filename entities
+
+In the context of Microscopy, a session ([`ses-<label>`](../99-appendices/09-entities.md#ses))
+can refer to all the acquisitions between the start and the end of an imaging experiment
+for ex vivo imaging or a subject lab visit for biopsy procedure and/or in vivo imaging.
+
+The [`sample-<label>`](../99-appendices/09-entities.md#sample) entity is REQUIRED for
+Microscopy data and is used to distinguish between different samples from the same subject.
+The label MUST be unique per subject and is RECOMMENDED to be unique throughout the dataset.
+
+<!--- The following example will be generated automatically with macros after community review. -->
+For example:
+Three brain slices (`sample-01` to `sample-03`) extracted from subject `sub-01`.
+```Text
+sub-01/
+    microscopy/
+        sub-01_sample-01_<modality_suffix>.<ext>
+        sub-01_sample-02_<modality_suffix>.<ext>
+        sub-01_sample-03_<modality_suffix>.<ext>
+        sub-01_<modality_suffix>.json
+```
+In this example, the JSON metadata is common for all samples of `sub-01`.
```

Comment:
this is getting into an ["Inheritance Principle"](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#the-inheritance-principle).  I think this fits it ok if we look at Examples 2 and 3 there, although I think we still have "Inheritance principle" not really formulated "unambiguously" (see https://github.com/bids-standard/bids-specification/issues/102) .

So - as such Inheritance principle applies to above as well, although the sentence below suggests that inheritance principle is only for the specification at higher level directories.  May be adjust wording a bit to make it clear that above is also follows from inheritance principle ?

Reply:
I'm sorry, I'm not sure to entirely understand your comment, what are "above" and "below" referring to?

We use that formulation twice, so I agree to reword to make it clearer if it's not.
One option would be to say:
"In this example, the JSON metadata is common for all samples of sub-01. JSON metadata may also be defined for each sample as appropriate, for example `sub-01_sample-01_<modality_suffix>.json`."

Reply:
Hi @yarikoptic!
I read the discussion in #102, and I'm not sure to understand completely the issue.
I made a suggestion above to change the text without referring to the inheritance principle. Please let me know if this wording is less ambiguous. Thank you!

Reply:
I think referring to inheritance principle is a good thing as long as it follows it.

> I'm sorry, I'm not sure to entirely understand your comment, what are "above" and "below" referring to?

By "above" (I think) I meant the "In this example, the JSON metadata is common for all samples of `sub-01`." sentence.  By "below" I just meant the "JSON metadata may be defined per subject or per sample as appropriate..." where you do explicitly point to inheritance principle (IP).  My point was that the "above" sentence might also be following IP, and thus wording could have been adjusted to not make "above" sentence sound like an ad-hoc custom statement. 
Checkout the rewrite of IP, and in particular the 2.3 rule (all 1s in the markdown): https://github.com/bids-standard/bids-specification/pull/946/files#diff-78a960934ce25feed2c29612b842d25131d0360556c2afc4dcaa9b60aceeb2baR550  -- I think it nicely applies here to that .json, thus IP applies even to that "above" case.

Reply:
Thank you for the explanation, I understand better your initial comment now.
The sentence "In this example, the JSON metadata is common for all samples of sub-01." was meant as an example of IP and is indeed following IP.

I would suggest to switch the order of the 2 sentences to make it clearer, something like this:
```
JSON metadata may be defined per subject or per sample as appropriate, as per the inheritance principle.
In the example above, the JSON metadata is common for all samples of "sub-01" and defined per subject.
```